Source/WebScheduler.Client.Core/Repositories/ScheduledTaskRepository.cs

```diff
@@ -197,9 +197,7 @@ FROM OrleansStorage
             WHERE  
                 TenantId = @TenantId
                 AND IsScheduledTaskDeleted = false
-                AND GrainTypeHash=2108290596
-            ORDER BY ScheduledTaskCreatedAt ASC
```

Comment:
Note on line 205 we don't set Pagination values nor do we need to order the count of the set.